app/mailers/hackathon/digest_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class Hackathon::DigestMailer < ApplicationMailer
+  def by_location
```

Comment:
The goal of `ByLocation` was to allow for other criteria in the feature, not to have a separate digest for it. Either we could move it to a `HackathonsMailer` and change this to `#digest` or just the latter.

Reply:
good catch!
[`4346285` (#76)](https://github.com/hackclub/hackathons-backend/pull/76/commits/434628534f83646b51ca1fcf2cad5274b2a645ef)